tests/minimax.wcon

```diff
@@ -0,0 +1,39 @@
+{
+    "tracker-commons":true,
+    "metadata":{
+           "lab":{"location":"CRB, room 5020", "name":"Behavioural Genomics" },
+           "who":"Firstname Lastname",
+           "timestamp":"2012-04-23T18:25:43.511Z",
+           "temperature":{ "experiment":22, "cultivation":20, "units":"C" },
+           "humidity":{ "value":40, "units":"%" },
+           "dish":{ "type":"petri", "size":35, "units":"mm" },
+           "food":"none",
+           "media":"agarose",
+           "sex":"hermaphrodite",
+           "stage":"adult",
+           "age":"18:25:43.511",
+           "strain":"CB4856",
+           "image_orientation":"imaged onto agar",
+           "protocol":"text description of protocol",
+           "software":{
+                "tracker":{ "name":"Software Name", "version":"1.3.0"},
+                "featureID":"@OMG"
+           },
+           "settings":"Any valid JSON entry with hardware and software configuration can go here"
+     },
+    "units":{"t":"s", "x":"mm", "y":"m"},
+    "data":[
+        {"@OMG":{
```

Comment:
Did the spec get rewritten to allow entries that don't have "id" etc.?  Because JSON does not require Objects to have keys in any particular order, there is no way to tell that this entry doesn't have "id" etc. without reading the whole thing.  What is the rule supposed to be?  Personally, I would put the whole "@OMG" section outside of "data" (which is okay by spec).